src/components/ec-inline-input-field/ec-inline-input-field.vue

```diff
@@ -23,6 +23,12 @@
         @edit="edit"
       />
     </template>
+
+    <ec-inline-input-field-copy
+      v-else-if="isCopiable"
```

Comment:
Is should be `true` if `isCopiable` is true and also if `value` is filled in. Both conditions are needed, if not, we'd see the button with an empty block
![image](https://user-images.githubusercontent.com/4802418/91187793-d8ad5880-e6f0-11ea-9ce2-46528194a0ef.png)